src/dali_backend.cc

```diff
@@ -224,6 +224,7 @@ TRITONSERVER_Error* TRITONBACKEND_ModelInstanceFinalize(TRITONBACKEND_ModelInsta
 TRITONSERVER_Error* TRITONBACKEND_ModelInstanceExecute(TRITONBACKEND_ModelInstance* instance,
                                                        TRITONBACKEND_Request** reqs,
                                                        const uint32_t request_count) {
+  TimeRange tr("[DALI BE] ModelInstanceExecute");
```

Comment:
I believe it's unnecessary at this stage - they are triggered only once and do not impact the processing too much. At least in DALI case. It might be a good idea to profile these functions as well, however the backend engine in Triton would be a way better place to do that - this would enable the profiling in all other backends as well.

Reply:
I agree, that they should be available inside TRITON itself. But we should add and keep them until TRITON adds support for NVTX markers.
While they are supposed to be executed only once I still think it is important to see if they are and how DALI behaves when TIROTN internal changes (maybe not everything looks like we think it does).

Reply:
Fair enough, I added those markers